src/ai/composite/property_handler.hpp

```diff
@@ -144,7 +144,7 @@ class vector_property_handler : public base_property_handler {
 		factory_(vec, cfg);
 	}
 private:
-	bool do_add(int pos, const config &cfg)
+	bool do_add(std::size_t pos, const config &cfg)
```

Comment:
This probably changes  behaviour  since the line below checks for `pos<0`

Reply:
Yeah, it looks like the CI caught it as well. I'm a bit surprised that works like that, but I suppose it does make sense.